benchmark/src/main/java/org/corfudb/benchmark/Main.java

```diff
@@ -0,0 +1,21 @@
+package org.corfudb.benchmark;
+
+import org.openjdk.jmh.annotations.Mode;
+import org.openjdk.jmh.runner.Runner;
+import org.openjdk.jmh.runner.RunnerException;
+import org.openjdk.jmh.runner.options.Options;
+import org.openjdk.jmh.runner.options.OptionsBuilder;
+
+public class Main {
+    public static void main(String[] args) throws RunnerException {
+        Options opt = new OptionsBuilder()
```

Comment:
Let's use docopt to take some arguments from the command line and provide some documentation.